docs/src/test/java/jdocs/future/FutureDocTest.java

```diff
@@ -85,4 +85,17 @@ public void useRetry() throws Exception {
 
     retriedFuture.toCompletableFuture().get(2, SECONDS);
   }
+
+  @Test
+  public void useRetryWithPredicate() throws Exception {
+    // #retry
+    Callable<CompletionStage<String>> attempt = () -> CompletableFuture.completedFuture("test");
+
+    CompletionStage<String> retriedFuture =
+        Patterns.retry(
+            attempt, (notUsed, e) -> e != null, 3, java.time.Duration.ofMillis(200), system);
```

Comment:
```suggestion
            attempt, (notUsed, e) -> e != null, 3, Duration.ofMillis(200), system);
```

Reply:
It's been used in doc, so better to keep the FQCN